docs/contributor/releases.md

```diff
@@ -0,0 +1,39 @@
+# Releases
+
+Claircore releases are cut as and when they are needed.
+
+Traditionally releases are always cut from the `main` branch so as not to complicate
+the release process, however, on rare occasions when a fix is time-sensitive then
+cutting a release from a release branch is possible.
+
+## Process
+
+> **_NOTE:_** Ensure changelog entries have been created for the relevant commits.
+> (see [Changelog documentation](./changelog.md))
+
+### From main
+
+```sh
+.github/scripts/prepare-release -b main -r upstream v1.x.x
+```
+
+Follow the `prepare-release` command's instructions to merge changelog updates and release the tag.
+
+### From release branch
+
+First create the relevant release branch (e.g if you are releasing `v1.6.1` create `release-1.6` from the
+previous tag (in this case `v1.6.0`)). Then backport any commits and push up the release branch.
+
+```sh
+git branch release-1.x v1.x.x
+git cherry-pick -x {commit sha}
+git push upstream release-1.x
+```
```

Comment:
```suggestion
```sh
LAST_MINOR=v0.999.0
BRANCH=release-${LAST_MINOR%.*} # e.g. release-v0.999
git branch $BRANCH $LAST_MINOR
TO_BACKPORT=beefc0ffee # Use the commit digest of the original commit
git cherry-pick -x $TO_BACKPORT
git push upstream $BRANCH
```
```